src/twisted/words/test/test_irc.py

```diff
@@ -9,13 +9,27 @@
 import operator
 import time
 
+from twisted.internet import protocol, task
```

Comment:
There are lots and lots of string constants in this module and it _might_ be helpful to have them be a consistent type between 2/3; a `from __future__ import unicode_literals` might catch potential issues.